packages/orchestration/src/examples/kitchen-sink.contract.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * @file Primarily a testing fixture, but also serves as an example of how to
+ *   leverage basic functionality of the Orchestration API with async-flow.
+ */
```

Comment:
Feedback welcome if anyone can think of a better name than `kitchen-sink`

Reply:
A kitchen sink will be some effort to maintain. Whenever we add some functionality we'll test it in a targeted fashion and have to also include it here.

If the scope is to test async flow, maybe "stress-flows" or "basic-flows"